packages/core/admin/admin/src/components/GuidedTour/utils/handleGuidedTourVisibility.js

```diff
@@ -0,0 +1,8 @@
+import { axiosInstance } from '../../../core/utils';
+
+export const handleGuidedTourVisibility = async setUserShouldSeeGuidedTour => {
```

Comment:
Could you move this api call to the AuthenticatedApp and rename it to fetchCurrentUser?